packages/bulkapi/src/controllers/BulkApi.ts

```diff
@@ -40,47 +39,50 @@ export class BulkApiImpl implements BulkApi {
         validator.validateSyntax()
         validator.validateReferences()
         if (validator.validationResult.hasErrors()) {
-            res.status(400)
-            res.send({ issues: [validator.validationResult.issues.map(issue => issue.errorMsg())] })
+            lionwebResponse(res, 400, {
+                success: false,
+                messages: validator.validationResult.issues.map(issue => ({kind: issue.issueType, message: issue.errorMsg()}))
+            })
         } else {
-            const issues: string[] = []
+            const issues: ResponseMessage[] = []
             for (const node of chunk.nodes) {
                 if (node.parent !== null && node.parent !== undefined) {
-                    res.status(400)
-                    issues.push(`Node ${node} cannot be created as partition because it has a parent.`)
+                    issues.push({ kind: "PartitionHasParent", message: `Node ${node} cannot be created as partition because it has a parent.`})
                 }
                 for (const containment of node.containments) {
                     if (containment.children.length !== 0) {
-                        issues.push(`Node ${node.id} cannot be created as a partition because it has children in containment ${containment.containment.key}`)
+                        issues.push({ kind: "PartitionHasChildren", message: `Node ${node.id} cannot be created as a partition because it has children in containment ${containment.containment.key}` })
```

Comment:
I think it would be useful to have all the constants indicating kinds of errors defined in a single place

Reply:
Done, but ... messages are not perse errors, Even when the call has success === `true`, there could be messages.